src/nodejs/src/audio-context-lazy.ts

```diff
@@ -0,0 +1,82 @@
+const LogScope = 'AudioContextLazy';
+
+export class AudioContextLazy {
+    private whenInitialized?: Promise<void> = null;
+    private audioContext?: AudioContext = null;
+
+    constructor() {
+        this.addInitEventListeners();
+    }
+
+    public async get(): Promise<AudioContext> {
+        if (!this.audioContext) {
+            await this.whenInitialized;
+        }
+
+        return this.audioContext;
+    }
+
+    private async initialize(): Promise<void> {
+        const audioContext = await this.create();
+        await this.warmup(audioContext);
+        this.audioContext = audioContext;
+    }
+
+    private async create() {
+        const audioContext = new AudioContext({
+            latencyHint: 'interactive',
+            sampleRate: 48000,
+        });
+
+        console.log(`${LogScope}.initialize: audioContext.state=`, audioContext.state);
+        await audioContext.resume();
```

Comment:
Are you sure you don't need anything like that 
```
    if (audioContext.state === 'suspended')
        await audioContext.resume();
```

Reply:
done